pkg/storaged/client.js

```diff
@@ -1069,11 +1069,17 @@ function stratis2_start() {
                                                                                    "/org/storage/stratis2",
                                                                                    { watch: false });
 
+                stratis.subscribe(
+                    {
+                        path_namespace: "/org/storage/stratis2",
+                        interface: "org.freedesktop.DBus.Properties",
+                        member: "PropertiesChanged",
+                        arg0: "org.storage.stratis2.filesystem"
+                    },
+                    (path, _iface, _signal, args) => stratis2_fixup_pool_notifications(path, args[1]));
```

Comment:
These 8 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18361-20230216-212122-6af132ab-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
These coverage messages happen because this code only runs on RHEL/CentOS 8 (stratis 2). Coverage runs on Fedora 37 which has stratis 3. So I think that's expected, and I'll close the conversations.